docs/platforms/dotnet/common/data-management/data-collected.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Data Collected
+description: "See what data is collected by the Sentry SDK."
+sidebar_order: 1
+---
+
+Sentry takes data privacy very seriously and has default settings in place that prioritize data safety, especially when it comes to personally identifiable information (PII) data. When you add the Sentry SDK to your application, you allow it to collect data and send it to Sentry during the runtime of your application.
+
+The category types and amount of data collected vary, depending on the integrations you've enabled in the Sentry SDK. Here's a list of data categories the Sentry .NET SDK collects:
+
+## HTTP Headers
+
+By default, the Sentry SDK sends HTTP headers to Sentry but redacts any headers that may contain sensitive data. (See the [list of headers](https://github.com/getsentry/sentry-dotnet/blob/main/src/Sentry/Internal/RedactedHeaders.cs#L7) that are redacted).
+
+## Cookies
+
+By default, the Sentry SDK doesn't send cookies.
+
+If you want to send cookies, set the option `SendDefaultPii = true` when initializing the Sentry SDK.
+
+A [data scrubber](/platforms/dotnet/data-management/sensitive-data/) can be used to filter information before sending data to Sentry.
```

Comment:
It's worth pointing out that Sentry will strip sensitive cookies in Relay before data hit any storage:

https://github.com/getsentry/relay/blob/249971e1702991c64cf0f7f89773fb019ffb4a95/relay-pii/src/convert.rs#L235-L263

Should we note this everywhere @stephanie-anderson